variables.tf

```diff
@@ -56,6 +61,18 @@ variable "aws_region" {
   default     = "us-east-1"
 }
 
+variable "cloudwatch_logs_sqs_queue_name" {
```

Comment:
It might be worth mentioning that this is also used as a base for the dead letter queue's name.

Reply:
Good idea.  I did that in https://github.com/cisagov/cool-sharedservices-cdm/pull/46/commits/8df490c3610a1e649c0dcd48d72c0422445064f2.